terragrunt/modules/docs-rs/builder.tf

```diff
@@ -32,7 +32,14 @@ resource "aws_iam_role_policy" "builder_s3" {
       // Access to s3
       {
         Effect = "Allow"
-        Action = "s3:*"
+        Action = [
+          "s3:PutObject",
+          "s3:GetObject",
```

Comment:
I *think* GetObject isn't necessary and it only needs ListBucket.

Reply:
yep, I'm quite certain we don't need `GetObject`

Reply:
Would [this](https://github.com/rust-lang/docs.rs/blob/d97bbf54b25362e1d1010f7fb6e29c5324cfb0a6/src/storage/s3.rs#L74-L79) not fail without?

Reply:
Removed this and it still seems to work.